ruff_lsp/server.py

```diff
@@ -53,6 +57,11 @@
 from typing_extensions import TypedDict
 
 from ruff_lsp import __version__, utils
+from ruff_lsp.utils import RunResult
+
+if os.environ.get("RUFF_LSP_DEBUG"):
+    log_file = Path(__file__).parent.parent.joinpath("pygls.log")
```

Comment:
Is this the standard logfile name when using python or what does `pygls` stand for?

Reply:
`pygls` is the lsp library we're using, the code code is just the example from https://pygls.readthedocs.io/en/latest/pages/advanced_usage.html?highlight=log#logging. I'm happy to change this to anything else if you have good ideas for logging, this was a very opportunistic addition because i sometimes need a log file that's independent of VS code

Reply:
Good point, renamed it `ruff-lsp.log`